sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/JsonWebKey.java

```diff
@@ -169,7 +169,7 @@ public String getId() {
      * @param keyId The kid value to set
```

Comment:
Since this is going to be a public API now, update the javadoc.
```suggestion
     * @param keyId The keyId value to set
```